### DISCUSSION TITLE
Video: going modular. Test loss explodes

### DISCUSSION THREAD

**Discussion by Neat-Snap** [mrdbourke/pytorch-deep-learning#1232] [Q&A]
<img width="612" height="311" alt="Screenshot 2025-07-28 at 15 01 08" src="https://github.com/user-attachments/assets/8f16b9f4-feb8-4ede-8176-d6536b23c318" />

Somewhy if I train the model for more epochs (I've tried for 30, 50 and 100) the test loss starts to explode while the training loss goes down. Test accuracy stays around 50% as well. I know that all of these are definetely not the best signs but I really don't know what to do...

Maybe someone has already faced the same issue?
